### PR TITLE
Upgrades the UI to Bootstrap 4

### PR DESCRIPTION
**Pull request by zeagord**
This PR is a WIP to upgrade the Zipkin UI to bootstrap-4. At least it will give us a sleek UI and may be a good start when we have bandwidth to rewrite the UI. 

I will limit the scope of this PR is only migrate to bootstrap-4 and fix minor things.

TODO:
* Prune unused dependencies
* Fix UI glitches

screen shot 2561-08-08 at 16 58 31
screen shot 2561-08-08 at 16 58 49
screen shot 2561-08-08 at 16 58 56
screen shot 2561-08-08 at 16 59 05

### CODE REVIEW

File: zipkin-ui/js/component_ui/infoPanel.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import {component} from 'flightjs';
+// import bootstrap // eslint-disable-line no-unused-vars
+//     from 'bootstrap-sass/assets/javascripts/bootstrap.js';
 import bootstrap // eslint-disable-line no-unused-vars
-    from 'bootstrap-sass/assets/javascripts/bootstrap.js';
```
Comment (codefromthecrypt) [inline zipkin-ui/js/component_ui/infoPanel.js]: does this or anything else mean we can drop any deps?

**Review by codefromthecrypt — COMMENTED** [review]
loving it! let us know when ready and I will test manually too.

**Comment by zeagord** [comment]
It's completed now.

File: zipkin-ui/css/trace.css [hunk 2]
```diff
@@ -203,7 +203,7 @@
   margin-right: 5px;
 }
 
-.save {
+/* .save {
```
Comment (shakuzen) [inline zipkin-ui/css/trace.css:206]: Can we get rid of this?

**Review by shakuzen — APPROVED** [review]
I won't pretend I understood most of those changes, but looks good as far as I can tell.

File: zipkin-ui/templates/layout.mustache [hunk 3]
```diff
@@ -1,42 +1,28 @@
-    <div id='navbar' class='navbar navbar-inverse' role='navigation'>
-      <div class='container'>
-        <div class='navbar-header'>
+  <nav class='navbar navbar-expand-lg navbar-dark bg-dark'>
           <a class='navbar-brand' href='{{contextRoot}}'>
-            Zipkin<span class='muted' style='font-size: .75em; padding-left: 10px;' data-i18n="nav.inves">Investigate system behavior</span>
+          <img src="https://zipkin.io/public/img/zipkin-logo-200x119.jpg" width="60" height="40" class="d-inline-block align-top rounded" alt=""/>
+            ZIPKIN <span class='font-weight-light navbar-text' style="font-size: .75em;" data-i18n="nav.inves">Investigate system behavior</span>
```
Comment (shakuzen) [inline zipkin-ui/templates/layout.mustache]: Does this change the `Zipkin` in the upper left of the UI to uppercase?

Comment (zeagord) [reply zipkin-ui/templates/layout.mustache]: Changed it to logo only

File: zipkin-ui/css/trace.css (hunk 2 above)
Comment (zeagord) [reply zipkin-ui/css/trace.css:206]: Removed it

**Comment by codefromthecrypt** [comment]
you are fantastic

**Pull request merged**